hive-metastore/src/main/java/org/apache/iceberg/hive/MetastoreUtil.java

```diff
@@ -73,8 +73,12 @@ public static void alterTable(
   }
 
   private static boolean detectHive3() {
+    return findClass(HIVE3_UNIQUE_CLASS);
```

Comment:
Do we still need this? Doesn't MetastoreClientVersion give us a Version 3 check?

Reply:
Good call. Removed.